lib/engine/game/g_18_fr/round/stock.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../../g_1817/round/stock'
+
+module Engine
+  module Game
+    module G18FR
+      module Round
+        class Stock < G1817::Round::Stock
+          def finish_round
+            # Do not move stock prices after Stock Round, they'll move after Share Redemption Round
+
+            @game.players.each do |player|
+              if player.shares.any? { |s| s.percent.negative? }
+                @game.player_sr_with_short_count[player] += 1
```

Comment:
You missed renaming the variable here:
```suggestion
                @game.extra_cert_limit[player] += 1
```